### PR TITLE
Plans: support free products in the JetpackProductCard

### PR DESCRIPTION
**Pull request by rcanepa**
#### Changes proposed in this Pull Request

* Add support for free products.
* Hardcode Jetpack CRM price at -1 (to enable the free price section).

#### Implementation notes

* I will address the technical debt in a follow-up.

#### Testing instructions

* Run this PR (any environment).
* Visit the Plans page.
* Verify that the Jetpack CRM card looks like the capture below.

Fixes 1196341175636977-as-1198189456928808

#### Demo
Kapture 2020-10-13 at 10 58 02

image
image
image

### CODE REVIEW

File: client/components/jetpack/card/jetpack-product-card-alt/index.tsx [hunk 1]
```diff
@@ -50,7 +50,8 @@ type OwnProps = {
 	isOwned?: boolean;
 	isDeprecated?: boolean;
 	expiryDate?: Moment;
-	hidePrice?: boolean;
```
Comment (rcanepa) [inline client/components/jetpack/card/jetpack-product-card-alt/index.tsx:53]: In this new version of the Plans grid, all products feature their prices, so we no longer need this property.

File: client/my-sites/plans-v2/product-card-alt/index.tsx [hunk 2]
```diff
@@ -74,6 +80,14 @@ const ProductCardAltWrapper = ( {
 		item?.monthlyProductSlug || ''
 	);
 
+	const isFree = originalPrice === -1 && discountedPrice === -1;
+	// TODO: we should move this text to where we keep translations for products and plans.
+	const isFreeMessage = [ PRODUCT_JETPACK_CRM, PRODUCT_JETPACK_CRM_MONTHLY ].includes(
+		item.productSlug
+	)
+		? translate( 'Start managing contacts now' )
+		: null;
```
Comment (rcanepa) [inline client/my-sites/plans-v2/product-card-alt/index.tsx:89]: This doesn't belong here. In a follow-up PR I'll make sure to put this in the right place.

File: client/my-sites/plans-v2/use-item-price.ts [hunk 3]
```diff
@@ -111,8 +111,8 @@ const useItemPrice = (
 
 	// Jetpack CRM price won't come from the API, so we need to hard-code it for now.
 	if ( item && [ PRODUCT_JETPACK_CRM, PRODUCT_JETPACK_CRM_MONTHLY ].includes( item.productSlug ) ) {
-		discountedPrice = 11;
-		originalPrice = 132;
+		discountedPrice = -1;
+		originalPrice = -1;
```
Comment (rcanepa) [inline client/my-sites/plans-v2/use-item-price.ts:115]: For now, -1 is going to mean that the product is free.

**Review by eeeeevon13 — CHANGES_REQUESTED** [review]
Please make the word free "semi bold" (it should match the thickest font in the price) also change the wording on the button to read "Start using CRM for free" 
Screen Shot 2020-10-12 at 4 24 29 PM

**Comment by rcanepa** [comment]
Thanks @eeeeevon13 ! I have updated the PR with the following changes:

- Make the Free word bolder (use the same font-weight the rest of the prices use).
- Update the CTA copy.

**Review by robertf4 — APPROVED** [review]
As long as there is a followup PR to address the mentioned technical debt, LGTM!

**Comment by rcanepa** [comment]
I actually managed to remove the tech debt right here in this PR.

**Review by eeeeevon13 — APPROVED** [review]
Looks good to me!

**Review by keoshi — CHANGES_REQUESTED** [review]
Requesting some changes before merging:

- `.plan-price-free__main` should have a `font-weight: 400` and not `600`.
- CRM card should be visible when monthly billing is selected, as mentioned in p1602593660073000-slack-CQXNTE9K9

Everything else looks fantastic!

**Comment by rcanepa** [comment]
@keoshi , I updated the PR with the following changes: 
- Decrease the `Free` word font-weight to 400.
- Display Jetpack CRM on monthly billing.

**Review by keoshi — APPROVED** [review]
Perfect, @rcanepa ! Thank you.

🚀

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4968765

Thank you @rcanepa for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4968765).

**Pull request merged**